packages/client/utils/draftjs/getTagsFromEntityMap.ts

```diff
@@ -4,8 +4,8 @@ const getTagsFromEntityMap = <T = string>(entityMap: RawDraftContentState['entit
   const entityKeys = Object.keys(entityMap)
   const tags = new Set<T>()
   for (let i = 0; i < entityKeys.length; i++) {
```

Comment:
nit: `Object.entries`

Reply:
As I see, `key` is not needed here, so will use `Object.values(entityMap).forEach((entity)`